Made a new goes page with updated information for new GOES E/W products.

tiffanycmeyer13 opened the pull request:
-Added a lot more RGB products all with images
-Updated the GLM area
-Added new section for DMW
-Added new section for vertical temp/moisture profiles
-Specified which sectors had derived products
-Updated all images

@srcarter3 I'm still working on updating the pqact section, I'll commit that later today, but let me know on the rest. Do you think I should add images to the GLM section?

srcarter3 commented on docs/cave/goes-satellite.md:
```diff
@@ -0,0 +1,308 @@
+# GOES 16/17
+
+The goesr EDEX decoder supports the ingest of GOES products coming over NOAAPort and Unidata's IDD. These include single channel imagery, derived products (Level 2b netCDF files), gridded Geostationary Lightning Mapper (GLM) products (produced by Eric Bruning at Texas Tech), CIRA created RGB specific products, and vertical temperature/moisture profiles. Using derived parameters, additional RGB an ...
```
I know the repo is called 'goesr', but I think for this paragraph, for human readability it should say "The GOES-R EDEX decoder...".
Also, the single channel imagery, derived products, glm, cira and profiles should have internal links to their corresponding sections below (for CIRA just link to the first product).

tiffanycmeyer13 replied:
I could color code/bold/quote the "goesr" - I was trying to use the actual name of the edex decoder that is being used - and similarly in the last paragraph the "dmw" EDEX decoder - that's the name of the EDEX decoder.

The paragraph below has all of the internal links (this was just how it was before....) but I can change it to be just in this first paragraph

srcarter3 commented on docs/cave/goes-satellite.md:
```diff
@@ -0,0 +1,308 @@
+* Vegetation (0.64 - 0.87 &mu;m)
+* Upper Level Info (11.2 - 6.19 &mu;m)
+
+!!! note "The rendering of these products uses the **Jep** package in Python, which has known to have issues installing on the MacOS and Windows versions."
```
I'm not sure I'd word it quite this way... Maybe say which need to be installed using very specific instructions found on our [installation page](link to our installation page).

tiffanycmeyer13 replied:
Sure, I only updated the "formatting" of the note, the wording was what existed previously. Do we have problems with jep in Mac now - I didn't think so? New wording:

"The rendering of these products uses the **Jep**  package in Python, which has specific installation steps (link to instructions) for Windows"

srcarter3 commented on docs/cave/goes-satellite.md:
```diff
@@ -0,0 +1,308 @@
+
+## Derived Products
+
+Derived products are also known as **Level 2+** products.  Currently there's only derived products from GOES East available in AWIPS. Each sector has a different set of products available. To find out some more information on some of the products please the [**Quick Guides**](http://rammb.cira.colostate.edu/training/visit/quick_guides/) compiled by CIRA.
```
"Currently there are only..."

srcarter3 commented on docs/cave/goes-satellite.md:
```diff
@@ -0,0 +1,308 @@
+
+!!! note "These may not all be available for each sector."
+
+The current products offered in CAVE are and to the right is which GOES East sector they are available for (F=Full Disk, C=CONUS, M=Mesoscale):
```
I think this sentence is incomplete?  "The current products offered in CAVE are ? and to the right...)

tiffanycmeyer13 replied:
okay, I'll fix it haha. thanks

srcarter3 reviewed (commented):
Just a couple small things.

tiffanycmeyer13 committed:
Made a new goes page with updated information for new GOES E/W products.
-Added a lot more RGB products all with images
-Updated the GLM area
-Added new section for DMW
-Added new section for vertical temp/moisture profiles
-Specified which sectors had derived products
-Updated all images
